c4p2n/models.py

```diff
@@ -23,7 +32,7 @@ class CallForPaperDefinition(BaseModel):
 
     id: str
     title: str
-    _fields: List[DefinitionField] = Field(None, alias="fields")
+    _fields: List[DefinitionField] = Field([], alias="fields")
```

Comment:
Not sure if we should use an empty list here because we always expect this field to be present in original data. My opinion is that we should use Ellipsis to indicate that this field is required and explicit declaration is used only to provide an alias.

Reply:
Fair point.